docs/release/role-handbooks/release-lead/README.md

```diff
@@ -0,0 +1,254 @@
+# Release Team Lead
+
+## Overview
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Responsibilities](#responsibilities)
+- [Tasks](#tasks)
+  - [Finalize release schedule and team](#finalize-release-schedule-and-team)
+  - [Add/remove release team members](#addremove-release-team-members)
+  - [Prepare main branch for development of the new release](#prepare-main-branch-for-development-of-the-new-release)
```

Comment:
Doesn't this TOC miss items which are there? e.g.
`Optional] Track Release and Improvement tasks`

Reply:
Thank you for pointing that out, I addressed this in my most recent commits.